packages/cadl-ranch-specs/http/azure/client-generator-core/usage/main.tsp

```diff
@@ -0,0 +1,67 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+import "@azure-tools/typespec-client-generator-core";
+
+using TypeSpec.Http;
+using Azure.ClientGenerator.Core;
+
+@doc("Test for internal decorator.")
+@supportedBy("dpg")
+@scenarioService("/azure/client-generator-core/usage")
+namespace _Specs_.Azure.ClientGenerator.Core.Usage;
+
+@scenario
+@scenarioDoc("""
+This scenario contains two public operations. Both should be generated and exported.
+The models are override to roundtrip, so they should be generated and exported as well.
+""")
```

Comment:
It has nothing to do about roundtrip.

A model of `input + output` could be just an input to op1, and output to op2. It does not need to be roundtrip at all (of course, roundtrip means `intput + output`, but not reverse).

Reply:
But if we add `@usage(Usage.input | Usage.output)` we will generate the model the same as a roundtrip model, right?

Reply:
No, that depends on whether emitter has additional process for roundtrip model.

If you test usage, not roundtrip, just don't mention roundtrip,